### PR TITLE
Added support for APIs managing SDDC

### PR DESCRIPTION
**Pull request by GaryJBlake**
Added Get-CloudBuilderSDDC
Added Start-CloudBuilderSDDC
Added Restart-CloudBuilderSDDC
Added Get-CloudBuilderSDDCValidation
Added Start-CloudBuilderSDDCValidation
Added Stop-CloudBuilderSDDCValidation
Added Restart-CloudBuilderSDDCValidation

### CODE REVIEW

File: docs/functions/SDDC (Cloud Builder)/Get-CloudBuilderSDDC.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Get-CloudBuilderSDDC
```
Comment (LifeOfBrianOC) [inline docs/functions/SDDC (Cloud Builder)/Get-CloudBuilderSDDC.md:1]: Intentionally not using Get-VCFCloudbuilder?

Comment (GaryJBlake) [reply docs/functions/SDDC (Cloud Builder)/Get-CloudBuilderSDDC.md:1]: Yes - I thought about using VCF in front but decided to not add it - Wanted to make it clear the cmdlts were different endpoints and this seemed like the best approach

Comment (alagoutte) [reply docs/functions/SDDC (Cloud Builder)/Get-CloudBuilderSDDC.md:1]: i think, it is always better to kept -VCFXXX, you known it is coming from PowerVCF...  (but only my opinion !)

File: powerVCF.psm1 [hunk 2]
```diff
@@ -4070,4 +4324,14 @@ Function Resolve-PSModule
     Return $searchResult
 }
 
+ dev-400-gjb-sddcs
```
Comment (LifeOfBrianOC) [inline powerVCF.psm1:4327]: This looks odd?

**Pull request merged**